src/pages/index.js

```diff
@@ -74,115 +88,266 @@ const closeModalOnEsc = (event) => {
 
 const closeModalOnOverlayClick = (event) => {
   const openedModal = document.querySelector(".modal_opened");
-  if (
-    openedModal &&
-    !openedModal
-      .querySelector(".modal__container, .modal__content")
-      .contains(event.target)
-  ) {
+  if (event.target.classList.contains("modal")) {
     closeModal(openedModal);
   }
 };
 
 // Event Listeners
-profileEditButton.addEventListener("click", () => {
-  profileNameInput.value = profileNameElement.textContent;
-  profileJobInput.value = profileJobElement.textContent;
-  openModal(editProfileModal);
+profileElements.editButton.addEventListener("click", () => {
+  profileElements.nameInput.value = profileElements.nameElement.textContent;
+  profileElements.jobInput.value = profileElements.jobElement.textContent;
+  openModal(profileElements.modal);
 });
 
-profileCloseButton.addEventListener("click", () => {
-  closeModal(editProfileModal);
+profileElements.closeButton.addEventListener("click", () => {
```

Comment:
**Could be improved**

You can make a universal handler for any close buttons.

It will look something like this:

```js
// Find all close buttons
const closeButtons = document.querySelectorAll('.modal__close');

closeButtons.forEach((button) => {
  // Find the closest popup only once
  const popup = button.closest('.modal');
  // Set the listener
  button.addEventListener('click', () => closePopup(popup));
});



```

That’s why we make universal css classes for similar elements: here it’s `modal__close` for any close button.

So now you can add 100 modals: their close buttons will be handled automatically.